### DISCUSSION TITLE
Offset außerhalb des Menüs drehbar?

### DISCUSSION THREAD

**Discussion by Braste** [Mogli12/KeyboardSteer#748] [Previous version]
Ich kann leider in der Tastaturbelegung keine Taste für das Drehen des Offsets finden und am Ende jeder Bahn ins Menü gehen zu müssen, nur um das Offset zu drehen, ist sehr nervig. 

- Übersehe ich die Belegung hierfür?
- Falls nicht, möchte ich hiermit ein Feature Request stellen.

**Comment by Mogli12** [comment]
Hi, das ist leider nicht vorgesehen. Die Idee ist, dass man einmalige Einstellungen über das Menü vornimmt.

**Comment by Braste** [comment]
Aber das Drehen des Offsets ist keine einmalige Einstellung, es muss je nach Gerät am Ende jeder Bahn gemacht werden.

**Comment by Mogli12** [comment]
Es gibt ja 2 Werte für Offset. Bei einem Drehpflug sollten ein Wert positiv, der andere negative sein. Beispiel: 1 und -1. Bei einem Mähwerk sollten die Werte gleich sein, also 1.5 und 1.5, Am Ende der Bahn sollte man so nichts tauschen müssen.
